R/misc.R

```diff
@@ -0,0 +1,41 @@
+
+#' Available panel types
+#' 
+#' Displays a data.frame that presents the list of available panel types,
+#' with their decoded and encoded names.
+#' 
+#' Decoded names are used to define the panels visible at initialization.
+#' Refer to the documentation of the \code{iSEE} function for more information.
+#' 
+#' Encoded names are used to name the individual HTML elements in the UI.
+#' Those names may be used to anchor tour steps at HTML elements using the \code{rintrojs} package.
+#'
+#' @param row.names Which column should be set as \code{row.names}.
+#' The default \code{NA} does not set rownames.
+#'
+#' @return A data.frame.
+#' @export
+#' 
+#' @author Kevin Rue-Albrecht
+#' 
+#' @seealso \code{\link{iSEE}}
+#'
+#' @examples
+#' availablePanelTypes()
+availablePanelTypes <- function(row.names=c(NA, "encoded", "decoded")){
+    
+    row.names <- match.arg(row.names)
+    
+    df <- data.frame(
+        decoded=panelTypes,
+        encoded=names(panelTypes),
```

Comment:
I don't see that this is necessary, the user should never see the encoded panel names AFAIK.

Reply:
Check out the Rd page. I was thinking about users/developers who want to write tours. Knowing the encoded panel names becomes necessary there.

Reply:
Can't we just return `panelTypes`, rather than a full fledged function?

Reply:
I've implemented your suggestion in b37d912e660ac9a7ff2f9c84b8f2fcd4bd00d172